test/vm.install

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
Please move this file to some subdirectory too.

Reply:
Done.